src/styles/components/_specification-block.scss

```diff
@@ -1,55 +1,28 @@
 .cmp-specifications-block {
-  display: flex;
-  flex-direction: column;
+  display: grid;
+  grid-template-columns: 1fr;
   gap: 3.75rem;
   line-height: 1.5625rem;
   letter-spacing: 0.02rem;
 
-  @media (min-width: $breakpoint-grid-large-screen-1115) {
+  @media (min-width: $breakpoint-specification-block-grid) {
     display: grid;
```

Comment:
We don't need to re-declare `display: grid;` here, since it's already set at the lower breakpoint.